lib/models/message_container.dart

```diff
@@ -0,0 +1,181 @@
+import 'package:flutter_opendroneid/models/constants.dart';
+import 'package:flutter_opendroneid/pigeon.dart' as pigeon;
+import 'package:dart_opendroneid/src/types.dart';
+import 'package:flutter_opendroneid/utils/compare_extension.dart';
+
+class MessageContainer {
```

Comment:
Please add a short description of this class' purpose (especially why it contains all the message types).

I actually think that this is how `MessagePack` should've been implemented in `dart-opendroneid` (each message type appearing once or not at all). Will check with @optical-o .